inc/dbmysql.class.php

```diff
@@ -482,4 +488,95 @@ public function getVersion(): string {
       $raw = $req['version()'];
       return $raw;
    }
+
+   /**
+    * Check if timezones are active.
+    *
+    * Under MySql, a command must be run to initilize data,
+    * and some privileges may be needed to read informations.
+    *
+    * @return boolean
+    */
+   public function areTimezonesActives() :bool {
+      //try to query TZs
+      $criteria = [
+         'COUNT'  => 'cpt',
+         'FROM'   => 'mysql.time_zone_name',
+      ];
+      $iterator = $this->request($criteria);
+      $result = $iterator->next();
+      if (!$result) {
+         Toolbox::logWarning('Access to timezone table (mysql.time_zone_name) is not allowed.');
+         return false;
+      }
+
+      if (count($iterator) == 0) {
```

Comment:
I think a COUNT operation will always return a row.
```suggestion
      if ($result['cpt'] == 0) {
```